demos/jobs/jobs_demo_mosquitto/jobs_demo_mosquitto.c

```diff
@@ -1200,12 +1214,9 @@ int main( int argc,
         errx( 1, "fatal error" );
     }
 
-    for( i = 0; i < ( sizeof( topics ) / sizeof( topics[ 0 ] ) ); i++ )
+    if( subscribe( h, JobsNextJobChanged ) == false )
     {
-        if( subscribe( h, topics[ i ] ) == false )
-        {
-            errx( 1, "fatal error" );
-        }
+        errx( 1, "fatal error" );
     }
 
     if( sendStartNext( h ) == false )
```

Comment:
sendStartNext is a POST api which scales separate from Describe Job with $next as the ID which is a GET api. sendStartNext transactions will compete with the Update api which might result in throttling at scale.

Reply:
The jobs library only uses MQTT.

Reply:
Apologies - I meant to say that the category of the API is for modification instead of retrieval. SendStartNext and DescribeJob are handled and scaled differently.

Reply:
I'll try to understand this better.  Any such changes will merit a new PR.